src/webextension/list/manage/containers/all-items.css

```diff
@@ -2,20 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.all-items {
-  border-inline-end: 1px solid #d7d7db;
-}
-
-.all-items > .item[data-selected] {
-  background-color: #ebF5ff;
-  border-bottom-color: #99bff2;
-  box-shadow: 0 -1px 0 #99bff2;
-}
-
-.all-items > .item:first-child[data-selected] {
-  border-top-color: #99bff2;
-}
-
 .all-items:focus > .item[data-selected] {
```

Comment:
Let's move this to `item-list.css` too. (This means we can delete this file entirely.)

Reply:
How can I target the container focus (which was .all-items) from within the `item-list.css` file? From within `item-list.js` I don't see so a way to add a class to target to the container.

Reply:
You need to pass `className={whatever}` to the `<ScrollingList/>`, and then do pretty much the same CSS as here.

Reply:
I tried adding a class that way but it doesn't seem to be bringing it in (I have also defined the style in `item-list.js`).

<img width="792" alt="screen shot 2018-03-08 at 11 06 07 am" src="https://user-images.githubusercontent.com/5185090/37168033-d51d9258-22c0-11e8-99c5-3bcd2a1069e6.png">

Reply:
Did you also remove the `className` from `all-items.js`? If you didn't, it'll override the `className` here (because it's getting pulled in from `{...props}`, which occurs later on this line).

Maybe I should establish a general rule that `{...props}` goes first to avoid this issue.

Reply:
👍 Got it, thanks for your help!